google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java

```diff
@@ -79,45 +79,96 @@
     /**
      * Returns the maximum allowed number of zones in the project.
      */
+    @Deprecated
     public int zones() {
+      return getZones();
+    }
+
+    /**
+     * Returns the maximum allowed number of zones in the project.
+     */
+    public int getZones() {
       return zones;
     }
 
     /**
      * Returns the maximum allowed number of records per {@link RecordSet}.
      */
+    @Deprecated
     public int resourceRecordsPerRrset() {
+      return getResourceRecordsPerRrset();
+    }
+
+    /**
+     * Returns the maximum allowed number of records per {@link RecordSet}.
+     */
+    public int getResourceRecordsPerRrset() {
       return resourceRecordsPerRrset;
     }
 
     /**
      * Returns the maximum allowed number of {@link RecordSet}s to add per {@link
      * ChangeRequest}.
      */
+    @Deprecated
     public int rrsetAdditionsPerChange() {
+      return getRrsetAdditionsPerChange();
+    }
+
+    /**
+     * Returns the maximum allowed number of {@link RecordSet}s to add per {@link
+     * ChangeRequest}.
+     */
+    public int getRrsetAdditionsPerChange() {
```

Comment:
While I kind of agree that rrset is unpleasant, naming was agreed with people from the DNS team so I believe we should just stick to it.